conda/recipes/librmm/recipe.yaml

```
@@ -0,0 +1,115 @@
+# Copyright (c) 2018-2025, NVIDIA CORPORATION.
+schema_version: 1
+
+
+# bashlet to use for this:
+# IFS=- read -r RAW_VERSION GIT_DESCRIBE_NUMBER GIT_DESCRIBE_HASH <<< $(git describe --tags)
+# Takes string like "v25.02.00a-49-g77dadb88" and splits on the `-` and drops the result into three variables
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  cuda_version: ${{ (env.get('RAPIDS_CUDA_VERSION') | split('.'))[:2] | join(".") }}
+  cuda_major: ${{ (env.get('RAPIDS_CUDA_VERSION') | split('.'))[0] }}
+  date_string: ${{ env.get("RAPIDS_DATE_STRING") }}
+  GIT_DESCRIBE_HASH: ${{ env.get("GIT_DESCRIBE_HASH") }}
+  GIT_DESCRIBE_NUMBER: ${{ env.get("GIT_DESCRIBE_NUMBER") }}
+
+recipe:
+  name: librmm-split
+
+cache:
+  source:
+    path: ../../..
+
+  requirements:
+    build:
+      - cmake ${{ cmake_version }}
+      - ninja
+      - make
+      - ${{ compiler('c') }}
+      - ${{ compiler('cxx') }}
+      - ${{ compiler('cuda') }} =${{ cuda_version }}
+      - cuda-version =${{ cuda_version }}
+      - ${{ stdlib("c") }}
+
+  build:
+    script:
+      file: build.sh
+      secrets:
+        - AWS_ACCESS_KEY_ID
+        - AWS_SECRET_ACCESS_KEY
+        - AWS_SESSION_TOKEN
+        - CMAKE_C_COMPILER_LAUNCHER
+        - CMAKE_CUDA_COMPILER_LAUNCHER
+        - CMAKE_CXX_COMPILER_LAUNCHER
+        - CMAKE_GENERATOR
+        - PARALLEL_LEVEL
+        - SCCACHE_BUCKET
+        - SCCACHE_IDLE_TIMEOUT
+        - SCCACHE_REGION
+        - SCCACHE_S3_USE_SSL
+        - SCCACHE_S3_NO_CREDENTIALS
+        - if: linux64
+          then:
+            - SCCACHE_S3_KEY_PREFIX=librmm-linux64
+        - if: aarch64
+          then:
+            - SCCACHE_S3_KEY_PREFIX=librmm-aarch64
+
+outputs:
+  - package:
+      name: librmm
+      version: ${{ version }}
+    build:
+      number: ${{ GIT_DESCRIBE_NUMBER }}
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ GIT_DESCRIBE_HASH }}_${{ GIT_DESCRIBE_NUMBER }}
+      script: install_librmm.sh
+    requirements:
+      host:
+        - cmake ${{ cmake_version }}
+        - cuda-version =${{ cuda_version }}
+        - fmt ${{ fmt_version }}
+        - spdlog ${{ spdlog_version }}
+      run:
+        - ${{ pin_compatible('cuda-version', upper_bound='x', lower_bound='x') }}
+        - ${{ "cudatoolkit" if cuda_major == "11" else "cuda-version" }}
```

Now that we are just using the v1 format, we could simplify [with (new style) selectors]( http://rattler.build/v0.31.1/selectors/#variant-selectors ) like so

```suggestion
        - if: cuda_major == "11"
          then: cudatoolkit
```